editor.txt

```diff
@@ -0,0 +1,107 @@
+Editor has two distinct modes accessed via command line parameters:
+- System editor used for star system editing
+- Model viewer used for model inspection
+Without any parameters, the editor will not start.
+	
+		System editor
+		~~~~~~~~~~~~
+		
+This is a gui tool for editing and authoring json formatted star systems.
+
+Usage:
+~~~~~~
+editor -system
+
+Mouse controls:
+~~~~~~~~
+<MMB> 			- rotate view
+<Mouse wheel> 	- zoom view
+<LMB>			- select body
+<double LMB>	- center on body
+<RMB>			- Edit context menu
+<shift>			- faster navigation
+<ctrl>			- slower navigation
+
+
+Menus:
+~~~~~~~~
+File - commonly used file operations
+	New system 	- create a blank system (ctrl+N)
+	Open file 	- open an existing system.json (ctrl+O)
+	Save		- save the system as a system.json (ctrl+S)
+	Save As 	- save the system to another name (ctrl+shift+S)
+	Quit		- Close the editor (ctrl+Q)
+Edit
+	Center on body 	- center view on selected body (double click)
+	Add Child		- Add a child body to the selected object (ctrl+A)
+	Add Sibling		- Add a sibling body to the selected object (ctrl+shift+A)
+	Delete Body		- Deletes the selected body (ctrl+W)
+	Sort Bodies		- Sort bodies in the outliner according to Semi-Mayor Axis
+Windows - misc tools and shortcut reference
+
+Outliner:
+~~~~~~~~
+The left side panel shows the hierarchical structure of the system. The list is collapsible, and bodies can be selected either here or in the system view itself. The order of bodies can be customized, but that will not affect their distance from their parent body.
+
+Right panel:
+~~~~~~~~
+You can edit the properties of the selected body in the panel on the right side. These are common astronomical and orbital parameters. For further information, the wikipedia article on orbital elements can be a good starting point.
+Some parameters allow for unit switching (km, AU, light seconds for example) for more intuitive editing.
+If the system from the outliner, its overall properties can be customized, such as its name, short and long description, galactic coordinates government, faction alignment.
+Systems are placed in a 8*8*8 lightyears sector of the galaxy, indicated by three integer coordinates, and its position is then preciesely placed within that sector via its position coordinates. The origin point of the sector coordinate system is Sol.
+
+
+
+		Model Viewer
+		~~~~~~~~~~~~
+
+This is a simple tool to view a game model using the game's rendering engine.
+It's main purpose is to test and debug models, and generally isn't for
+end-user use.
+
+Usage:
+~~~~~~
+editor -mv <model>
+
+<model>		- name of the model defined in lua
+
+	Example:
+	./editor -mv interdictor
```

Comment:
For symmetry, I'd suggest 
```
    Example:
    ./editor --modelviewer
	./editor -mv interdictor
```

Come to think of it, does passing model name as argument work anymore? (just tried with `Dome1`, and `cargo` to no avail).

Reply:
Easier to change documentation than the code :)

(Classic example: "map is wrong, the giant rock is on the wrong side of the road" - move rock to where map said)

Reply:
You should be able to run `editor --modelviewer=<modelname>`, supporting the `editor -mv <modelname>` case is something that I can restore support for.